src/components/Articles.jsx

```diff
@@ -20,7 +23,7 @@ export default function Articles() {
         setError(err);
         setIsLoading(false);
       });
-  }, []);
+  }, [articleSortQuery]);
```

Comment:
There is a delay everytime I change the articleSortQuery. Can you make the loading component appear while I'm waiting for the new response from the server?